src/apps/explorer/components/OrderWidget/index.tsx

```diff
@@ -29,11 +29,16 @@ export const OrderWidget: React.FC = () => {
   }, [orderId])
 
   // TODO: this is just for testing. The hooks will not be here
-  const { value: buyToken } = useErc20({ address: order?.buyToken, networkId: 4 })
-  const { value: sellToken } = useErc20({ address: order?.sellToken, networkId: 4 })
-
-  console.log(`buy token`, buyToken)
-  console.log(`sell token`, sellToken)
+  // const { value: buyToken } = useErc20({ address: order?.buyToken, networkId: 4 })
+  // const { value: sellToken } = useErc20({ address: order?.sellToken, networkId: 4 })
+  // console.log(`buy token`, buyToken)
+  // console.log(`sell token`, sellToken)
+
+  const multiple = useMultipleErc20({
```

Comment:
this is also, just for testing the hook too, right? u will kill it after?

Reply:
Yep, only for testing